src/ui/chat-window.js

```diff
@@ -487,10 +487,45 @@ export class ChatScene extends Phaser.Scene {
                                 }
                                 break;
 
-                            // TODO:
                             case "unlock-level":
+                                // Usage:
```

Comment:
Now that we’re adding more commands I think we should move it to its own file

Reply:
https://github.com/EthanVieira/rs-clicker/pull/74/commits/98c58434bea66c2ef7f1366c77bc1018de0c8cff